carbon-design-system/carbon
# Introduce experimental grid support

## Overview

Adds support for the experimental grid into carbon.

### Added

* `src/globals/grid/experimental` directory added with experimental grid specs

### Changed

* `src/globals/grid/_grid.scss` moved to `src/globals/grid/classic`
* `src/global/grid/_grid.scss` exports the grid module and toggles imports for experimental

### Removed


## Testing / Reviewing

- Verify that by default the CSS outputs doesn't include experimental code
- Verify that setting `css--use-experimental-grid` to true includes the default experimental grid code
- Verify that setting `css--use-experimental-grid` and `css--use-experimental-grid-fallback` both to true includes the fallback for the experimental grid in addition to the default implementation
